apps/internal/base/base.go

```diff
@@ -86,6 +86,7 @@ type AuthResult struct {
 	Account        shared.Account
 	IDToken        accesstokens.IDToken
 	AccessToken    string
+	RefreshIn      time.Time
```

Comment:
I agree with adding this to the public API, but for consistency with `ExpiresOn` it should be added as `RefreshOn`

Reply:
Changed it to `refreshOn`